doc/arc/adr_006.md

```diff
@@ -1,21 +1,23 @@
-# TITLE (short noun phrase)
+# Sharding & Routing per Geographic Region
 
-Date: DATE
+Date: Nov 1 2020
```

Comment:
Silly, but let's be consistent w/ date format

Reply:
```suggestion
Date: 11/1
```